api/src/main/java/org/pmiops/workbench/notebooks/NotebooksServiceImpl.java

```diff
@@ -18,26 +21,45 @@
 
 @Service
 public class NotebooksServiceImpl implements NotebooksService {
+  private static final String CLUSTER_LABEL_AOU = "all-of-us";
+  private static final String CLUSTER_LABEL_CREATED_BY = "created-by";
+  private static final String DEFAULT_CLUSTER_NAME = "all-of-us";
+
+
   private static final Logger log = Logger.getLogger(NotebooksServiceImpl.class.getName());
 
   private final Provider<ClusterApi> clusterApiProvider;
   private final Provider<NotebooksApi> notebooksApiProvider;
   private final Provider<JupyterApi> jupyterApiProvider;
+  private final Provider<WorkbenchConfig> workbenchConfigProvider;
 
   @Autowired
   public NotebooksServiceImpl(Provider<ClusterApi> clusterApiProvider,
       Provider<NotebooksApi> notebooksApiProvider,
-      Provider<JupyterApi> jupyterApiProvider) {
+      Provider<JupyterApi> jupyterApiProvider,
+      Provider<WorkbenchConfig> workbenchConfigProvider) {
```

Comment:
You could probably just inject Provider<User> here instead of passing user email in. (Either way seems OK to me.)

Reply:
I was thinking about that, and wasn't sure it made sense to have the user provider here. Although I'm not really sure. Either way.